subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -954,6 +955,8 @@ private boolean isSettingsFileAvailable() {
 
         if (useOwnUserHomeServices || (gradleUserHomeDir != null && !gradleUserHomeDir.equals(buildContext.getGradleUserHomeDir()))) {
             properties.put(REUSE_USER_HOME_SERVICES, "false");
+        } else {
```

Comment:
:x: `useOwnUserHomeServices` is orthogonal, it should be complied with in all cases
from https://github.com/gradle/gradle/pull/6673/files#r215973768